smartsim/_core/control/jobmanager.py

```diff
@@ -242,7 +242,7 @@ def get_status(
                 job: Job = self[entity.name]  # locked
                 return job.status
 
-            return SmartSimStatus.STATUS_NEVER_STARTED
+            return FailedToFetchStatus.STATUS_NEVER_STARTED
```

Comment:
I like the change to FiledToFetchStatus.  Is "never started" most correct, or is it just something more like "status not found"